components/camel-couchbase/src/main/java/org/apache/camel/component/couchbase/CouchbaseConsumer.java

```diff
@@ -87,7 +87,7 @@ protected void doInit() {
 
         String rangeStartKey = endpoint.getRangeStartKey();
         String rangeEndKey = endpoint.getRangeEndKey();
-        if ("".equals(rangeStartKey) || "".equals(rangeEndKey)) {
+        if (rangeStartKey.isEmpty() || rangeEndKey.isEmpty()) {
```

Comment:
Warning this will throw an NPE if `rangeStartKey` or `rangeEndKey` is null

Reply:
True, but are nulls possible in them? I ask because previously the code would let two nulls (not the case if only one of them is null) into viewOptions in the line below:
viewOptions.startKey(rangeEndKey).endKey(rangeEndKey);  

Will add null checks anyway